results/fragments/creators.html

```diff
@@ -0,0 +1,4 @@
+<ul class="creator_list">
```

Comment:
Super-nit: throughout, my instinct would be to call this test_authors rather than creators, but I don't think it's user-visible, so up to you.

Reply:
I was motivated by the fact that the DC term used is 'creator' and not authors. That is why I systematically used the term 'creator' in the code for different variables. I would prefer to leave it that way...